ios/Openid4vpBle/Openid4vpBle.swift

```diff
@@ -48,16 +52,14 @@ class Openid4vpBle: RCTEventEmitter {
 
     @objc(setTuvaliVersion:)
     func setTuvaliVersion(version: String) -> String{
-//        tuvaliVersion = version
-//        os_log("Tuvali version - %{public}@",tuvaliVersion);
-//        Central.shared.tuvaliVersion = tuvaliVersion
+        tuvaliVersion = version
+        os_log("Tuvali version - %{public}@",tuvaliVersion);
         return tuvaliVersion
       }
 
     @objc(destroyConnection:)
     func destroyConnection(withCallback callback: @escaping RCTResponseSenderBlock) -> Any {
-        Wallet.shared.destroyConnection(isSelfDisconnect: true)
-        callback([])
+        wallet?.handleDestroyConnection(isSelfDisconnect: true)
```

Comment:
Once the destroy is done, reset the wallet variable to `nil`

Reply:
done with suggested changes.